src/app/state/transaction/saga.ts

```diff
@@ -257,6 +257,13 @@ export function* submitParaTimeTransaction(runtime: Runtime, amount: string, rec
 
   yield* call(OasisTransaction.signParaTime, chainContext, signer as Signer, tw)
   yield* call(OasisTransaction.submit, nic, tw)
+  yield* put(
+    transactionActions.transactionSent({
```

Comment:
what does this part do? it looks like it creates a transaction record of a "transfer." but this is after doing a paratime deposit. and the commit message is "refresh account balance." and the balance won't update immediately

Reply:
Added  a dedicated action for paraTime transfer to trigger refresh in account and wallet sagas.

Yes, balance won't update immediately that's why there is a delay effect. It's not ideal, but it's acceptable. We don't support WebSocket or polling to handle such case better.